homeassistant/components/glances/config_flow.py

```diff
@@ -0,0 +1,124 @@
+"""Config flow for Glances Client."""
+import glances_api
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_SCAN_INTERVAL,
+    CONF_SSL,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+)
+from homeassistant.core import callback
+
+from . import GlancesClient
+from .const import (
+    CONF_VERSION,
+    DEFAULT_HOST,
+    DEFAULT_NAME,
+    DEFAULT_PORT,
+    DEFAULT_VERSION,
+    DOMAIN,
+    SUPPORTED_VERSIONS,
+)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_NAME, default=DEFAULT_NAME): str,
```

Comment:
Why do we need to configure name?

Reply:
My idea is to be able to configure multiple Glances instances, not just one. In this case the name will be used when naming the sensors created.

Reply:
Can't we default to host name and let the user config names via the entity registry?

Reply:
Yeah I was thinking about the ip address. Maybe that's ugly.

I'm ok with this. :+1: